### PR TITLE
Rapid crossbow device: more gun than arrow

### PR DESCRIPTION
**Pull request by Noobetski**
## About The Pull Request
<details>
<summary>
	Moves RXD to guild and reworks it to be more of a ballistic counterpart to myrmidon. I tried to balance around the fact that it doesn't needs to be drawn anymore, but the numbers probably need to be looked at. Tested on localhost and it works. I didn't touch the other crossbow because it works fine already, only needs balance changes.
</summary>
<hr>

	
<hr>
</details>

## Changelog
:cl:

tweak: Reworked RXD. Hopefully it's more useful now.

/:cl:

### CODE REVIEW

**Comment by cdb-is-not-good** [comment]
Why'd you get rid of the drawing back mechanic? Sorta eliminates a part of the weapons uniqueness.

**Comment by Noobetski** [comment]
explained in guild channel on discord

**Comment by cdb-is-not-good** [comment]
> explained in guild channel on discord

Not everyone who's looking at the git will know what you're talking about. 

Tl;dr, because the whole gun got reworked into a child of another item and had all its code scrapped
